src/main/java/com/google/sps/servlets/InsertRepDatastoreServlet.java

```diff
@@ -55,7 +56,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
             throw new ServletException("Error: " + e.getMessage(), e);
         }
         if (rep == null) {
-            DatastoreManager.insertRepresentativeInDatastore(repName, title, username, password);   
+            List<Long> tabIds = DatastoreManager.insertTabsInDatastore(
+                new ArrayList<String> (Arrays.asList(repName.replaceAll("\\s+","") + "Other")), 
```

Comment:
I think you don't need to pass it into an ArrayList constructor. Type inference should do the trick :) are you seeing issues if you don't?

Also add a comment on what the replaceAll is for